### PR TITLE
Malkorok Script 

### PR DESCRIPTION
**Pull request by canewadar123**
Scripted Malkorok a little bit.

### CODE REVIEW

**Comment by cooler-SAI** [comment]
@cybrann, can you take a look please

**Comment by cybrann** [comment]
Well I will test this, it looks nice. Less work for me to do on malkorok :D

File: src/server/scripts/Pandaria/SiegeOfOrgrimmar/boss_malkorok.cpp [hunk 1]
```diff
@@ -1,130 +1,251 @@
...
+		{
+			DoCastToAllHostilePlayers(SPELL_ACIENT_MIASMA);
+			Talk(MALKOROK_AGGRO);
+			std::list<Player*> pl_list;
+			me->GetPlayerListInGrid(pl_list, 500.0f);
+			for (auto itr : pl_list)
+			{
+				itr->RemoveAura(SPELL_ACIENT_MIASMA);
+			}
+		}
```
Comment (cybrann) [inline src/server/scripts/Pandaria/SiegeOfOrgrimmar/boss_malkorok.cpp]: Hmm why would you make boss cast ancient misasma on death and then remove it immediately?
If it causes bug for players that did not have it, that would have been easily checked in for loop by if (itr->HasAura(SPELL_ANCIENT_MIASMA))

**Comment by cybrann** [comment]
Ok I will merge this and start working on it, it will save me some time to script boss.

**Pull request merged**